### PR TITLE
CSS: remove property normalization

### PR DESCRIPTION
**Pull request by mikesherov**

### CODE REVIEW

File: util/css.js [hunk 1]
```diff
@@ -36,35 +35,7 @@ define(function() {
...
-  // Prepopulate map with the only exception 'float'
-  styleProperty.memo = { float: 'cssFloat' };
-  styleProperty.pattern = /-([a-z])/g;
-  styleProperty.replacement = function(match, p1) {
-    return p1.toLocaleUpperCase();
-  };
-
-
   return {
-    extract: function(element) {
```
Comment (kaicataldo) [inline util/css.js]: Is this being removed because it's not being used anywhere?

Comment (mikesherov) [inline util/css.js]: Yes.

**Comment by kaicataldo** [comment]
:+1:

**Comment by mmase** [comment]
LGTM

**Pull request merged**